src/renderer/components/modals/BackupContents.tsx

```diff
@@ -47,11 +42,6 @@ export const BackupContents = (props: ModalContentsProps) => {
 		FlyModal.onRequestClose();
 	};
 
-	const onClickEditIgnore = async (site: Site) => {
-		const ignoreFilePath = await getIgnoreFilePath(site);
-		shell.openPath(ignoreFilePath);
```

Comment:
If we need to keep the same behavior as before, we could [call ipcRenderer.invoke](https://github.com/getflywheel/flywheel-local/pull/1937#discussion_r1492889799).

Reply:
Nice suggestion, thanks. Done in https://github.com/getflywheel/local-addon-backups/pull/101/commits/06dc69e4e716e05888cc43dbd4595845b7bfe1bf.

Reply:
Thanks, Nick! https://github.com/getflywheel/local-addon-backups/commit/06dc69e4e716e05888cc43dbd4595845b7bfe1bf looks great.